util/types/mydecimal.go

```diff
@@ -41,6 +44,11 @@ const (
 
 	MaxFraction = 30
 	DivFracIncr = 4
+
+	// HalfEven rounds normally.
+	HalfEven RoundMode = "HalfEven"
+	// Truncate just truncates the decimal.
+	Truncate RoundMode = "Truncate"
```

Comment:
Since ceiling mode is not supported, I think it is better not to expose Ceiling as a mode here.

WDYT @coocood